build/build.fsx

```diff
@@ -89,6 +94,25 @@ Target "Package" (fun _ ->
             })
 )
 
+Description("Run all benchmarks. Must be run with configuration=Release.")
+Target "Benchmarks" (fun _ ->
+    if configuration <> "Release" then
+        failwith "Benchmarks can only be run in Release mode. Please re-run the build with configuration=Release."
+
+    let benchmarkCsproj = "tests/NSubstitute.Benchmarks/NSubstitute.Benchmarks.csproj" |> Path.GetFullPath
+    let allBenchmarks = "*"
```

Comment:
should we be able to pass this as a build script parameter?

Reply:
Yes! But I have no idea how to do this 😆 . I'll read through the FAKE docs and see what I can find. 😄